engines/agi/text.cpp

```diff
@@ -274,7 +275,26 @@ void TextMgr::displayTextInsideWindow(const char *textPtr, int16 windowRow, int1
 	charPos_Pop();
 }
 
+Common::String rightAlign(Common::String line, va_list args) {
+	uint width = va_arg(args, uint);
+
+	while (line.size() < width)
+		line = " " + line;
+	return line;
+}
+
 void TextMgr::displayText(const char *textPtr, bool disabledLook) {
+	Common::String textString = Common::String(textPtr);
```

Comment:
No need to initialize the string if it is not manipulated. Needless copy.

But notice the scope must not change.

```
Common::String textString;
if (_vm->isLanguageRTL()) {
    textString = textPtr;
    ...
}

Reply:
OK.